<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeqBootModel</class>
 <widget class="QWidget" name="SeqBootModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="bootstrapCheckBox">
       <property name="text">
        <string>Bootstapped consensus tree</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Number of replicates</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="repsSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Seed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="seedSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>32765</number>
       </property>
       <property name="value">
        <number>99</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Consensus type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ConsModeComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Fraction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="FractionSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
